frontend/src/lib/components/popups/FilterEntry.svelte

```diff
@@ -106,7 +114,7 @@
 				<input
 					type="text"
 					bind:value={predicate.value}
-					class="h-8 border border-grey-lighter rounded"
+					class="h-[38px] pl-2 border border-grey-lighter rounded"
```

Comment:
I think generally it is good practice. In this case I had to make it match the other heights I can't set. Generally can try to avoid them, don't think it's the worst design pattern though.